src/ServiceManagement/RemoteApp/Commands.RemoteApp/packages.config

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Hyak.Common" version="1.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common" version="2.0.2" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Authentication" version="1.0.10-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Common.Extensions" version="0.13.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.RemoteApp" version="1.0.7" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.14.1-preview" targetFramework="net45" />
+  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Async" version="1.0.168" targetFramework="net45" />
+  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net45" />
+  <package id="Microsoft.Data.Edm" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.Data.OData" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.Data.Services.Client" version="5.6.2" targetFramework="net45" />
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="3.0.110281957-alpha" targetFramework="net45" />
```

Comment:
Please match the version to the same version used elsewhere in the project - this is a major version later than the version currently in use